lib/ansible/modules/system/ufw.py

```diff
@@ -235,8 +237,12 @@ def main():
         ),
         supports_check_mode=True,
         mutually_exclusive=[
-            ['app', 'proto', 'logging']
+            ['app', 'proto', 'logging'],
         ],
+        required_one_of=([command_keys]),
+        required_by=dict(
+            interface=('direction'),
```

Comment:
```suggestion
            interface=('direction', ),
```
Otherwise it is simply a string :)

Reply:
Not really, as I supported this case (a common issue).

https://github.com/ansible/ansible/pull/28662/files#diff-90085fdcec6ed8b273ba885eaee60328R1678